t/024_boolean_header_fields.t

```diff
@@ -2,6 +2,10 @@ use Test::More;
 use strict;
 use warnings;
 
+plan skip_all => "this test has not been converted as there's public admin on the new server";
```

Comment:
it's on the list, i just have to update the puppet to allow remote admin

but i'll be honest, it terrifies me! i'm sure this is exactly the kind of
thing you don't expose to the world!

On 27 October 2015 at 15:40, Michael D. Stemle, Jr. <
notifications@github.com> wrote:

> In t/024_boolean_header_fields.t
> <https://github.com/net-amqp-rabbitmq/net-amqp-rabbitmq/pull/86#discussion_r43139589>
> :
>
> > @@ -2,6 +2,10 @@ use Test::More;
> >  use strict;
> >  use warnings;
> >
> > +plan skip_all => "this test has not been converted as there's public admin on the new server";
>
> I don't like that we're skipping this since it had been a problem in the
> past. We should probably get this working again soon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/net-amqp-rabbitmq/net-amqp-rabbitmq/pull/86/files#r43139589>
> .
>